.github/workflows/github-actions-deploy.yml

```diff
@@ -0,0 +1,45 @@
+# got info from https://dev.to/knowbee/how-to-setup-continuous-deployment-of-a-website-on-a-vps-using-github-actions-54im
+name: Build & Deploy
+
+on:
+  push:
+    branches: [ main, develop, feature/docker ]
+  workflow_dispatch:
+
+jobs:
+  build_and_push:
+    name: Build & Push to DockerHub
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Docker Build and Push to DockerHub
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: robskt/rentmycar:latest
+
+  deploy_to_vps:
+    runs-on: ubuntu-latest
+    needs: build_and_push
+    name: Deploy to VPS
+    steps:
+      - name: Connect with SSH and Deploy
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.SSH_HOST }}
+          username: ${{ secrets.SSH_USER }}
+          key: ${{ secrets.SSH_KEY }}
+          script: |
+            cd /home/rob/rentmycar
```

Comment:
Gaat dit problemen opleveren als wij iets mergen?

Reply:
nee hoor :) Hij build en deployed alleen als we mergen naar main.

Reply:
Waar zoekt hij dan naar `/home/rob`?